lib/pharos/host/ubuntu/scripts/reset.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+
+rm -rf /etc/kubernetes/manifests
+sleep 5
+systemctl stop kubelet
+systemctl disable kubelet
+
+if which docker ; then
+    docker rm -fv $(docker ps -a -q)
+    systemctl stop docker
+    systemctl disable docker
+elif which crictl ; then
+    systemctl stop crio
+    systemctl disable crio
+    crictl rm $(crictl ps -a -q)
+fi
+
+kubeadm reset
+
+apt-get purge -y --allow-change-held-packages --purge kubeadm kubelet kubectl docker.io cri-o-${CRIO_VERSION}
```

Comment:
`apt-get purge 'cri-o-*'` should also work

Reply:
Good catch, will fix this.